src/main.cpp

```diff
@@ -8421,35 +8434,6 @@ bool MemorizeMessage(const CTransaction &tx, double dAmount, std::string sRecipi
                     sMessageValue="";
               }
 
-              if (sMessageType=="beacon" && sMessageAction=="A")
```

Comment:
Is the removal of this not an issue now that we verify the beacon when accepting the blocks?

Reply:
I do not think so. By removing this just memorizes every beacon. V9 beacon transaction rules do prevent new invalid beacons.